### ISSUE TITLE
Support For linenums for CodeHilite

### ISSUE THREAD

**Issue by sak96** [WnP/vimwiki_markdown#16]
line number can be enabled by  passing [linenums](https://python-markdown.github.io/extensions/code_hilite/#usage) to codehilite

```python
Codehilite(linenums=True)
```

If i want to raise PR which approach is advised.

1. create `VIMWIKIMARKDOWN_LINENUMS` for passing this arguement
2. parse `VIMWIKIMARKDOWN_EXTENSION` using `ast.literal_eval` and then add default plugins if they are not available

**Comment by WnP** [comment]
Hi @sak96,

That's a great idea. I prefer the first option by the way. Fell free to open a PR, I'll review it ASAP.

Thanks

**Comment by sak96** [comment]
I have raised pull request for both of them. #17 #18
do have a look at the doc links. ast.lteral_eval does not allow arbitrary code execution.
it only allows literals like `1 True None {}` and so on and is more flexible

**Comment by sak96** [comment]
#18 fixes this issue

**Comment by sak96** [comment]
@WnP Please close this issue when you release to pypi

**Comment by sak96** [comment]
```vim
let $VIMWIKI_MARKDOWN_EXTENSIONS='[{"codehilite": {"linenums": True}}]'
```

**Comment by sak96** [comment]
@WnP hi sorry for anoyying you again. But would you be able to release with this feature ?

**Comment by sak96** [comment]
@WnP, i am guessing you are pretty busy.
Closing the issue and installing using 
```bash
pip install git+https://github.com/WnP/vimwiki_markdown
```

**Comment by WnP** [comment]
Hi @sak96,

I've just released [0.4.0](https://pypi.org/project/vimwiki-markdown/0.4.0/).

Thanks for your interest.
